geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,73 @@
+import geomstats.geometry.spd_matrices as spd
```

Comment:
Yayy, the sampling module! 🎉 

This file needs an explanatory docstring at the top.

Imports should be in alphabetical order.